src/hooks/useConnectorWithTagDetail.ts

```diff
@@ -3,13 +3,13 @@ import {
     CONNECTOR_RECOMMENDED,
     TABLES,
 } from 'services/supabase';
-import { OpenGraph } from 'types';
+import { ENTITY_WITH_CREATE, OpenGraph } from 'types';
 import { useQuery, useSelect } from './supabase-swr';
 
 export interface ConnectorWithTagDetailQuery {
     connector_tags: {
         documentation_url: string;
-        protocol: string;
+        protocol: ENTITY_WITH_CREATE;
```

Comment:
Every time I encounter `protocol` I engage in an internal debate regarding the merit of exposing the term within the UI. Each time, I conclude that _entity_ is the more familiar and exposed term (as far as documentation is concerned) but I always revisit the argument. _Thinking out loud_.